go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
 	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.1.0
+	github.com/Backblaze/blazer v0.6.1
```

Comment:
Please revert all changes to go.mod _except_ for the blazer library update. Updating the other library belongs into a separate PR.

Reply:
Is this desired? The changes I included here were just the results of a `go mod tidy`. The other library updates are because of `blazer` now including a [`go.mod` file](https://github.com/Backblaze/blazer/blob/efa10c0fffe98ce85c0a927a13249cc695a4f8e5/go.mod), and requiring those newer versions.

Reply:
Oh indeed you're right. I'd have prefer to update those dependencies separately, but we can also update them right now.